src/components/GlobalLinterMessages/index.tsx

```diff
@@ -0,0 +1,33 @@
+import * as React from 'react';
+
+import { getCodeLineAnchorID } from '../CodeView/utils';
+import { scrollToSelectedLine as _scrollToSelectedLine } from '../CodeView';
+
+type PublicProps = {
+  className?: string;
+  children: React.ReactNode[];
+  scrollToSelectedLine?: typeof _scrollToSelectedLine;
```

Comment:
The spec needs a test to make sure that the ref prop gets added to the container. Unfortunately, you'll need to use `mount()` for that (but just for that one test). Here's an example:

https://github.com/mozilla/addons-code-manager/blob/da479dba41330c2714f4d9a722c3f29945dd8f1a/src/components/CodeView/index.spec.tsx#L249-L256